### ISSUE TITLE
Test reorganization

### ISSUE THREAD

**Issue by aucampia** [RDFLib/rdflib#1831]
@gjhiggins is busy reorganing tests. Creating this to track the target layout so that we are all clear on what we are moving to.

**Comment by aucampia** [comment]
@gjhiggins as discussed in #450 - I indicated for test suite data, this would be sensible `test/data/suites`, to me `test/consistent_test_data/test_suites` is quite redudant, as it contains test 3 times and consistent should somewhat be implied, but inside `test/data/` we could pottentially have `test/data/well-known/` for something like the TBL card.

**Comment by gjhiggins** [comment]
> this would be sensible `test/data/suites`

Agreed. In process of implementing, PR later today.

**Comment by gjhiggins** [comment]
Lots of changes, how to you want to handle this: https://github.com/gjhiggins/rdflib/tree/test-reorg?

**Comment by aucampia** [comment]
It looks like most of the changes is just moving of files with 3 exceptions, so I would just go ahead and make a PR for it, I will double check it though but if most of it is just moving files it is easy to review.

Just for my own reference, to compare: https://github.com/RDFLib/rdflib/compare/master...gjhiggins:test-reorg

**Comment by aucampia** [comment]
Okay I see github is not showing all changes, I made a fairly clean diff, minus all the moves: https://gist.github.com/aucampia/21671bccb12975d4d951156d6ffafa20

**Comment by aucampia** [comment]
@gjhiggins some of the changes seem to be just syncing the test files, maybe doing those in a separate PR will be slightly cleaner, but I am fine with the PR, I would go ahead and make it even if it is big.

**Comment by aucampia** [comment]
Yes, I would just go ahead with it, I can fairly easily review it I think, will just confirm and rule out remote files and then all that is left is our files and checking that the tests before and after are the same. Looks fairly good overall.

**Comment by gjhiggins** [comment]
> Yes, I would just go ahead with it 
Will do.

> checking that the tests before and after are the same
I did check that and they are.

All the changes to files are (should be) re-referencing the data file locations, apart from restoring a test of the otherwise-untested broken_parse_data n3 parse fails - which still fail.

**Comment by aucampia** [comment]
This should be solved before further re-org: https://github.com/RDFLib/rdflib/issues/1839
